### ISSUE TITLE
Remove dependency on structuremap*-signed

### ISSUE THREAD

**Issue by Sebbe** [markeverard/Chief2moro.ImageDataExtensions#11]
With episerver 11 addons should have a dependency on EPiServer.ServiceLocation.StructureMap >= and not have a dependency on structuremap anymore

**Comment by mohsinali1317** [comment]
Any status on this one?

**Comment by GeekInTheNorth** [comment]
I've just had a legacy code base come to me for an upgrade assessment.  Since this is still an issue, should we consider this library out of support?

**Comment by markeverard** [comment]
I guess the real question is, was this ever in support? 

I've no current intentions to do anything further with this. The source code is available here and released with an MIT license.

**Comment by GeekInTheNorth** [comment]
Thank you for responding. If needed, would you be happy for this repository to be forked and updated to work with the latest version of EPiServer/Optimizely?

**Comment by markeverard** [comment]
Yes absolutely. That's what the MIT license is all about :) Happy for you to fork and add a PR and then I'll publish the package on the Optimizely feed
